messages/en.json

```diff
@@ -2,12 +2,49 @@
   "title": "MABP Brand Website EN",
   "description": "Best Brand Website in the world!",
 
-  "Index": {
-    "title": "Hello world!"
-  },
   "navbar": {
     "home": "Home",
     "aboutus": "About us",
     "blog": "Blog"
+  },
+  "Header": {
+    "title": "From stellar websites to unique products, we deliver ",
+    "accent_title": "out-of-this-world solutions",
+    "contact_us": "Contact us",
```

Comment:
you are using two different variable names format camelCase and snake-case, we should choose one and apply for all. From what I saw on the doc of next-intl in the [example ](https://next-intl-docs.vercel.app/examples) they are using camelCase format

Reply:
Done!